cranelift-codegen/src/isa/x86/binemit.rs

```diff
@@ -101,6 +116,20 @@ fn put_rexop2<CS: CodeSink + ?Sized>(bits: u16, rex: u8, sink: &mut CS) {
     sink.put1(bits as u8);
 }
 
+/// Emit two-byte opcode: 0F XX with inferred REX prefix.
+fn put_dynrexop2<CS: CodeSink + ?Sized>(bits: u16, rex: u8, sink: &mut CS) {
+    debug_assert_eq!(
+        bits & 0x0f00,
+        0x0400,
```

Comment:
Doc says 0F (as emitted), yet compared with 04.

Reply:
That assertion is copied from 'put_rexopt2()`. These assertions should definitely not open-code the bit-patterns, but should be changed to use the new `EncodingBits` struct.